javascript/extractor/src/com/semmle/js/extractor/FileExtractor.java

```diff
@@ -42,7 +42,68 @@
       Pattern.compile("^(?s)\\s*\\{\\s*\"([^\"]|\\\\.)*\"\\s*:.*");
 
   /** The charset for decoding UTF-8 strings. */
-  private static final Charset UTF8_CHARSET = Charset.forName("UTF-8");
+  private static final Charset UTF8_CHARSET = StandardCharsets.UTF_8;
```

Comment:
Hmm. Wouldn't it be cleaner just to inline `UTF8_CHARSET` now that it isn't a method call any more?